weather10kb.js

```diff
@@ -25,18 +25,21 @@ function Weather10kbRequest(request) {
           apiKey: process.env.GOOGLE_API_KEY,
           formatter: null         // 'gpx', 'string', ...
         });
+        var countryCode = 'US';
         geocoder.geocode(request.params.location)
           .then(function(res) {
             if (res.length) {
               request.params.latitude = res[0].latitude
               request.params.longitude = res[0].longitude
               request.params.formatted_location = res[0].formattedAddress
+              request.params.locationSearch = request.params.location;
+              countryCode = res[0].countryCode;
             } else {
               // TODO throw exception?
               request.params.latitude = 0;
               request.params.longitude = 0;
             }
-            resolve();
+            resolve(countryCode);
```

Comment:
Excellent point. I suggest falling back to the default code in case the value is invalid. If you agree, I will add the missing check.